files/default/solr-restore-from-backup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+. `dirname $0`/solr-env.sh
+
+LATEST_BACKUP=$(aws s3 ls "s3://$BUCKET/solr_backup/$CORE_NAME/" |awk '{print $4}' |sort |tail -1)
```

Comment:
need a wiki page on this as well as how to pick an older backup if the original is 'bad'

Reply:
Added more info to https://ssq-qol.atlassian.net/wiki/spaces/QOL/pages/20381840/OpenData+Publications+CKAN+Rebuild+Index+commands+cheat+sheet